### PR TITLE
Data: Fahad Ali

### PR DESCRIPTION
**Pull request by imf-ali**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5245"></a>

### CODE REVIEW

File: data/imf-ali.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    "AWS",
+    "MERN Developer",
+    "NextJS",
+    "Spring Boot"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/imf_ali" },
+    { "icon": "FaGithub", "url": "https://github.com/imf-ali" },
+    { "icon": "FaInstagram", "url": "https://www.instagram.com/imf_ali" },
+    { "icon": "FaLink", "url": "https://imf-ali.netlify.app/" }
```
Comment (priyankarpal) [inline data/imf-ali.json:17]: @imf-ali I think you need to change the URL of your website. It's not working.

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

File: config/icons.json [hunk 2]
```diff
@@ -21,6 +21,7 @@
   "FaTwitter": "#00ACEE",
   "FaVimeo": "#1AB7EA",
   "FaYoutube": "#FF0000",
+  "FaRocketchat" : "#FF0000",
```
Comment (stevenovak123) [inline config/icons.json:24]: This should be a separate git commit and I am pretty sure you should raise an issue for this first and get approval for the same

**Review by stevenovak123 — CHANGES_REQUESTED** [review]
`data/imf-ali.json` LGTM. Maintainers will take a look into `icons`

File: config/icons.json (hunk 2 above)
Comment (imf-ali) [reply config/icons.json]: I have created the issue for the same

**Comment by imf-ali** [comment]
I have created an issue #5247. Please look into the same @stevenovak123.

**Review by iamthecloverly — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Comment by imf-ali** [comment]
Hey @kumarsonsoff3 can u look into this issue #5247.

**Pull request merged**